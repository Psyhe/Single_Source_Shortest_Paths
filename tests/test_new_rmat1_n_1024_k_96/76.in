1024 824 833
824 8 164
824 40 200
824 1 54
825 2 240
825 72 249
826 272 2
826 21 203
828 66 35
828 336 8
828 20 53
829 24 117
832 450 20
832 64 140
832 52 90
832 99 110
832 194 252
832 272 242
832 2 7
832 624 122
832 321 114
832 16 109
832 99 120
832 129 105
832 265 186
832 768 2
832 560 251
832 521 59
832 4 228
832 544 80
832 266 17
832 160 213
832 1 84
832 961 22
832 644 200
832 32 23
832 64 45
832 37 21
832 102 247
832 36 161
832 388 146
832 550 142
832 128 172
832 550 177
832 20 254
832 48 76
832 280 23
832 570 232
832 20 5
832 3 161
833 80 139
833 257 38
833 20 217
833 2 99
833 32 191
833 12 242
833 7 241
833 130 233
