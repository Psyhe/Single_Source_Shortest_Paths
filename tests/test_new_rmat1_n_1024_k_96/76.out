56
245
7
1e+18
16
122
1e+18
1e+18
9
43
